### PR TITLE
Formatting / Unsupported features / Small edits

### LINKED ISSUE #742: Update comments documentation
https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema#comments

Background: https://prisma-company.slack.com/archives/CEYCG2MCN/p1596528653196100?thread_ts=1596471041.195700&cid=CEYCG2MCN

### LINKED ISSUE #1405: Replace `https://codeload.github.com/...` urls with `https://pris.ly/quickstart`
Found in a few places https://github.com/prisma/docs/search?q=codeload.github.com
It makes the snippet nicer.

Screen Shot 2021-03-10 at 18 37 03

### LINKED ISSUE #1558: 'How do I achieve X with Prisma Migrate' landing page + links to schema docs

### LINKED ISSUE #1658: Extend dbgenerated doc
https://prisma-company.slack.com/archives/C5Z9TH6N9/p1620287002054000?thread_ts=1620201600.040200&cid=C5Z9TH6N9

### LINKED ISSUE #1662: Database-level functions with Prisma Migrate
https://prisma-company.slack.com/archives/C5Z9TH6N9/p1620295816064600?thread_ts=1620201600.040200&cid=C5Z9TH6N9

### LINKED ISSUE #1668: Document formatting rules
https://www.prisma.io/docs/concepts/components/prisma-schema#auto-formatting links to the specs repo, which is archived and not updated at all. We should pull that content into the documentation: https://github.com/prisma/specs/tree/master/schema#formatting-rules

### LINKED ISSUE #1669: Rewrite Auto Formatting section for Prisma schema
https://www.prisma.io/docs/concepts/components/prisma-schema#auto-formatting

Currently pretty weird, should get link to documentation of `prisma format` and VSCode extension.

Related to https://github.com/prisma/docs/issues/1668

### LINKED ISSUE #1677: Reference, Schema, Data Source, url, "Learn more" link is broken
https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#datasource

> Connection URL including authentication info. Most connectors use the syntax provided by the database, learn more.

"Learn more" links to https://www.prisma.io/docs/reference/api-reference/

### LINKED ISSUE #1687: Connection string --> Connection URL

### PR DESCRIPTION
**Pull request by mhwelander**

### CODE REVIEW

File: content/200-concepts/100-components/01-prisma-schema/07-features-without-psl-equivalent.mdx [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+```prisma highlight=3;normal
+model Star {
+  id           Int                      @id @default(autoincrement())
+  position     Unsupported("circle")?   @default(dbgenerated("'<(10,4),11>'::circle"))
+}
+```
+
+The `prisma migrate dev` and `prisma db push` command will both create a `position` field of type `circle` in the database. However, the field will not be available in the generated Prisma Client.
+
+## Unsupported features
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/07-features-without-psl-equivalent.mdx]: Maybe:
```suggestion
## Unsupported database features
```

File: content/200-concepts/100-components/01-prisma-schema/index.mdx [hunk 2]
```diff
@@ -297,22 +299,98 @@ model User {
...
+/// Customer node.
 
 model Customer {}
 ```
 
 ## Auto formatting
 
-Similar to tools like [gofmt](https://golang.org/cmd/gofmt/) and [prettier](https://github.com/prettier/prettier), PSL syntax ships with a formatter for
-`.prisma` files. The formatter can be enabled in the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma).
+PSL syntax ships with a formatter for `.prisma` files. There are two ways to format `.prisma` files:
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/index.mdx]:
```suggestion
Prisma supports formatting `.prisma` files automatically. There are two ways to format `.prisma` files:
```

File: content/200-concepts/100-components/01-prisma-schema/index.mdx [hunk 3]
```diff
@@ -297,22 +299,98 @@ model User {
...
+
+* Run the [`prisma format`](../../../reference/api-reference/command-reference#forma) <span class="api"></span> command.
+* Install the [Prisma VS Code extension](https://marketplace.visualstudio.com/items?itemName=Prisma.prisma) and invoke the [VS Code format action](https://code.visualstudio.com/docs/editor/codebasics#_formatting) - manually or on save.
+
+Like `gofmt` and unlike `prettier`, there are no configuration options - [formatting rules](#formatting-rules) are fixed.
```
Comment (janpio) [inline content/200-concepts/100-components/01-prisma-schema/index.mdx]:
```suggestion
There are no configuration options - [formatting rules](#formatting-rules) are fixed (similar to Golang's `gofmt` but unlike Javascript's `prettier`):
```

**Review by janpio — APPROVED** [review]
Great as always.

File: content/400-reference/200-api-reference/100-prisma-schema-reference.mdx [hunk 4]
```diff
@@ -1825,6 +1825,12 @@ Generate a globally unique identifier based on the [UUID](https://en.wikipedia.o
 - Compatible with `String`
 - Implemented by Prisma and therefore not "visible" in the underlying database schema. You can still use `uuid()` when using [introspection](../../concepts/components/introspection) by [manually changing your Prisma schema](../../concepts/components/prisma-client/working-with-prismaclient/use-custom-model-and-field-names) and [generating Prisma Client](../../concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client), in that case the values will be generated by Prisma's [query engine](../../concepts/components/prisma-engines/query-engine).
 
+<Admonition type='info'>
```
Comment (molebox) [inline content/400-reference/200-api-reference/100-prisma-schema-reference.mdx:1828]: TIL what this word means 😀

File: content/200-concepts/100-components/01-prisma-schema/07-features-without-psl-equivalent.mdx [hunk 5]
```diff
@@ -0,0 +1,131 @@
...
+    ADD PRIMARY KEY ("id");
+    ```
+
+1. Apply the migration:
+
+   ```terminal
+   npx prisma migrate dev
+   ```
+
+Each time you reset the database or add a new member to your team, all required functions are part of the migration history.
```
[the remark this answers is not in the document]
Comment (mhwelander) [reply content/200-concepts/100-components/01-prisma-schema/07-features-without-psl-equivalent.mdx]: Moved to its own page in migrate guides

**Pull request merged**